Application/Features/BookingFeature/Commands/CreateBookingCommand.cs

```diff
@@ -58,8 +58,8 @@ public async Task<CreateBookingCommandResponse> Handle(CreateBookingCommandReque
                 throw new NotFoundException($"There is no WorkPlace with id={request.WorkplaceId}");
             }
 
-            EnsureWorkplaceIsFree(request.WorkplaceId, request.StartDate, request.EndDate);
-            EnsureUserHasNotBookingThisTime(request.TelegramId, request.StartDate, request.EndDate);
+            await EnsureWorkplaceIsFree(request.WorkplaceId, request.StartDate, request.EndDate);
```

Comment:
Name this method as EnsureWorkplaceIsFreeAsync

Reply:
done